baselines/adafactor/pytorch/submission.py

```diff
@@ -0,0 +1,312 @@
+"""Submission file for Adafactor in PyTorch."""
+
+from functools import partial
+from typing import Dict, Iterator, List, Tuple
+
+from absl import logging
+import torch
+import torch.distributed.nn as dist_nn
+from torch.optim.lr_scheduler import CosineAnnealingLR
+from torch.optim.lr_scheduler import LinearLR
+from torch.optim.lr_scheduler import SequentialLR
+
+from algorithmic_efficiency import spec
+from algorithmic_efficiency.pytorch_utils import pytorch_setup
+
+USE_PYTORCH_DDP = pytorch_setup()[0]
+
+
+def init_optimizer_state(workload: spec.Workload,
+                         model_params: spec.ParameterContainer,
+                         model_state: spec.ModelAuxiliaryState,
+                         hyperparameters: spec.Hyperparameters,
+                         rng: spec.RandomState) -> spec.OptimizerState:
+  """Creates an Adafactor optimizer and a learning rate schedule."""
+  del model_state
+  del rng
+
+  # Create optimizer.
+  optimizer_state = {
+      'optimizer':
+          Adafactor(
+              model_params.parameters(),
+              lr=hyperparameters.learning_rate,
+              beta1=hyperparameters.beta1,
```

Comment:
the hparam will be called `one_minus_beta1`, and then pass it in as `beta1=hyperparameters.one_minus_beta1,`